core/src/services/split-view.js

```diff
@@ -223,6 +223,11 @@ class SplitViewSvcClass {
           IframeHelpers.getIframeContainer().style.paddingBottom = '';
           SplitViewSvc.sendMessageToClients('close.ok');
         });
+    } else {
+      if (comp.get().mfSplitView && comp.get().mfSplitView.settings && comp.get().mfSplitView.settings.onlyExpandBoxVisible) {
+        comp.get().mfSplitView.displayed = false;
+        comp.set({ mfSplitView: comp.get().mfSplitView });
```

Comment:
It seems `comp.get().mfSplitView` repeated a lot, I guess @UlianaMunich expects the code below:
```suggestion
      let splitViewState = comp.get().mfSplitView;
      if (splitViewState && splitViewState.settings && splitViewState.settings.onlyExpandBoxVisible) {
        splitViewState.displayed = false;
        comp.set({ mfSplitView: splitViewState });
```
Please double-check if it works.

Reply:
yeah, this is what I was suggesting above. A small cleanup to reduce the redundancy and increase the readability. Hope it makes sense

Reply:
Yes, good point @UlianaMunich. It definitely make sense... just didn't get your point :)